Dockerfile

```diff
@@ -18,8 +18,7 @@ RUN gop build -o ./community
 # Run
 FROM ubuntu:22.04
 
-RUN apk update && apk upgrade
-RUN apk --no-cache add libc6-compat
+RUN apt-get update
```

Comment:
It seems like nothing is installed. In that case, can we delete this command?

Reply:
Yes,I have deleted